src/Compilers/CSharp/Portable/Symbols/Source/SourceMethodSymbolWithAttributes.cs

```diff
@@ -610,6 +632,20 @@ private void DecodeWellKnownAttributeAppliedToMethod(ref DecodeWellKnownAttribut
             {
                 DecodeInterceptsLocationAttribute(arguments);
             }
+            else if (attribute.IsTargetAttribute(AttributeDescription.OverloadResolutionPriorityAttribute))
+            {
+                MessageID.IDS_OverloadResolutionPriority.CheckFeatureAvailability(diagnostics, arguments.AttributeSyntaxOpt);
+
+                if (IsOverride)
```

Comment:
nit: consider moving this case inside the `if (!CanHaveOverloadResolutionPriority)` case. They are really the same case, just with a refinement of the diagnostic ID. 
Maybe even consider `var errorCode = IsOverride ? ErrorCode.ERR_CannotApplyOverloadResolutionPriorityToOverride : ErrorCode.ERR_CannotApplyOverloadResolutionPriorityToMember;`